### DISCUSSION TITLE
Custom wallet

### DISCUSSION THREAD

**Discussion by NORAVemp** [rainbow-me/rainbowkit#1886] [Q&A] [answered]
I now have a plug-in wallet clownWallet. I downloaded it and successfully created it. When I use the custom wallet of the @rainbow-me/rainbowkit framework, clicking the call has no effect. Is there any configuration I am missing? Or do I have to submit my wallet to walletConnect in order to customize a wallet that is not supported by the @rainbow-me/rainbowkit framework?
<img width="976" alt="截屏2024-03-25 17 35 10" src="https://github.com/rainbow-me/rainbowkit/assets/96608866/09d67ae4-b736-400b-a944-b76bad2c744e">

**Comment by NORAVemp** [comment]
Please help me answer it @magiziz

**Reply by 0xmkh** [reply]
@NORAVemp Yes you'd need to create your own wallet in order to use it. Can you specify all the necessary mobile links including `qrCode` as well [here](https://github.com/rainbow-me/rainbowkit/blob/main/packages/rainbowkit/src/wallets/walletConnectors/argentWallet/argentWallet.ts) ?

**Reply by NORAVemp** [reply, marked as answer]
magiziz,thank you for your help
